gnd/src/main/java/com/google/android/gnd/ui/home/featuredetails/FeatureDetailsFragment.java

```diff
@@ -14,52 +14,49 @@
  * limitations under the License.
  */
 
-package com.google.android.gnd.ui.home.featuresheet;
+package com.google.android.gnd.ui.home.featuredetails;
 
 import android.os.Bundle;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.ImageView;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.core.view.WindowInsetsCompat;
 import butterknife.BindView;
 import com.google.android.gnd.MainViewModel;
 import com.google.android.gnd.R;
-import com.google.android.gnd.databinding.FeatureSheetFragBinding;
+import com.google.android.gnd.databinding.FeatureDetailsFragBinding;
 import com.google.android.gnd.ui.common.AbstractFragment;
-import com.google.android.gnd.ui.home.FeatureSheetState;
+import com.google.android.gnd.ui.home.BottomSheetState;
 import com.google.android.gnd.ui.home.HomeScreenViewModel;
 import javax.inject.Inject;
 
-public class FeatureSheetFragment extends AbstractFragment {
-
-  @BindView(R.id.feature_header_icon)
-  ImageView featureHeaderIcon;
+public class FeatureDetailsFragment extends AbstractFragment {
```

Comment:
Add comment explaining what `FeatureDetails` or reference another class so it's easy to find out what object this `Fragment` is representing

Reply:
Adding JavaDoc here. In a follow-up PR, I hope to either move the header into this fragment, in which case its existence would be more reasonably justified. (It currently only contains the observation list).